src/main/java/de/teamlapen/vampirism/potion/PotionBloodLoss.java

```diff
@@ -0,0 +1,17 @@
+package de.teamlapen.vampirism.potion;
+
+
+import net.minecraft.entity.SharedMonsterAttributes;
+
+
+
+public class PotionBloodLoss extends VampirismPotion {
+    public static PotionFeeding POTION= new PotionFeeding("Blood Loss", false, Integer.parseInt("FF0000", 16));
```

Comment:
I am surprised that it is working :D
Potions should be registered to Forge and their names are supposed to be lowercase and one word. 
However, it looks like it is not enforces right now. Still probably better to be prepared for future changes.

The other Potions are instantiated in `ModPotions#registerPotions`. If you add a `public static final Potion name = getNull();` at the top of the ModPotions class with exactly the same name as the potion name. It will be automatically filled with the instance by Forge magic.

Reply:
tbh i'm not sure why either of those still exist.  They're both literally reskinned slowness.  I think I was intending to tie the overlay to it?  I might as well just use the normal slowness effect.